docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -191,6 +191,40 @@ type BuildApiInterfacesJsonOptions = BuildInterfacesCommonOptions & {
   interfacesWithDedicatedPage: DocumentedInterfaces;
 };
 
+export interface InterfaceApiContent {
+  /**
+   * The name of the documented interface.
+   */
+  name: string;
+  /**
+   * The array of way to import this interface.
+   */
+  imports: string[];
+  /**
+   * The HTML content of the demonstrations list.
+   */
+  demos?: string;
+  /**
+   * The mapping of property name to their typing.
+   */
+  properties: {
+    [property: string]: {
+      /**
+       * The initial type definition.
+       */
+      type: { description: string };
+      default?: string;
+      required?: true;
+      isProPlan?: true;
```

Comment:
Is there a reason not to have a `plan?: 'pro' | 'premium' | 'community'` here?

Reply:
No other reason than "that's what the script returns"

I tried to do the update, but that would require a PR on the core repo to update properties section component

Reply:
Would probably make sense to update it at some point, but it's out of the scope of this PR then :+1: